### ISSUE TITLE
Regarding the Public Release of the SARDet-100K Testset

### ISSUE THREAD

**Issue by zcablii** [zcablii/SARDet_100K#2]
We have currently only released the training and validation sets of the SARDet-100K dataset, with the test set remaining unreleased and scheduled to be made public in September of this year. This decision is primarily due to our plans to organize an international/national algorithm competition, which will use the test set of this dataset for scoring. Therefore, to ensure the fairness of the competition, the test set will not be disclosed before the end of the event. We will make the test set publicly available as soon as the competition concludes, and we ask for your understanding in this matter. Meanwhile, the provided training and validation sets are sufficient to support the development and evaluation of research algorithms, and we hope they will aid in your research endeavours.

**Comment by yuhao20** [comment]
作者你好，请问可以开源一下关于文中处理后的DOTA数据集吗，或者能否描述一下具体处理前使用的是哪一版DOTA数据呢？
Hello, can you open-source the DOTA dataset after processing in the article, or can you describe which version of DOTA data is used before processing?

**Comment by zcablii** [comment]
You can refer to my [another repo](https://github.com/zcablii/LSKNet/tree/main/tools/data/dota) for the DOTA dataset preparation.

**Comment by Harmony521** [comment]
你好,目前不公开test,请问可以进行在线评估吗?

**Comment by zcablii** [comment]
We greatly appreciate your interest. We are planning to host an algorithm competition from May to September this year, and an online evaluation of the test set will be initiated in the later stages of the competition. To ensure the fairness of the competition, we are currently unable to disclose or evaluate the test set publicly online. We hope for your understanding in this matter. 

If you wish to verify and compare your models/algorithms, you may do so using the validation set that we provide. Along with sharing the weight files, we have also made public the training logs and the performance of our models on the validation set after 12 epochs of training. Comparing performance on the validation set should be sufficient for research purposes. Additionally, if you are interested, you can continue to follow our announcements and participate in the upcoming competition, and you will also be able to utilize our online test set evaluation during the competition.

**Comment by xiayifan1024** [comment]
考虑到训练数据集有许多来自其他数据集，未公开的测试数据是否包括这些数据集，这是否会造成后续比赛测试数据的泄露？

**Comment by zcablii** [comment]
> 考虑到训练数据集有许多来自其他数据集，未公开的测试数据是否包括这些数据集，这是否会造成后续比赛测试数据的泄露？


我们考虑到了这一点。所以我们在比赛中把图片做了随机重命名。而且，比赛要求中已说明“除了比赛提供的数据集，参赛队伍能且仅能使用ImageNet、 COCO和 DOTA-v1.0数据集作为预训练或者数据增广辅助。为维护竞赛的公平性和竞赛目标的纯粹性，严禁使用其他数据集。 ”
并且，我们会最终对参赛选手的提交代码和模型做训练与测试的复现。

**Comment by zcablii** [comment]
# 🔥🔥 PRCV2024 SARDet目标检测竞赛开赛啦 🔥🔥

大规模合成孔径雷达图像多类别目标检测竞赛来啦！大赛使用了本工作的SARDet-100K数据集！南开大学[VCIP](https://mmcheng.net/)课题组和CSIG机器视觉专委会、深势科技在PRCV2024联合举办了SARDet目标检测竞赛，本赛道不仅提供实时排行，开箱即用的baseline、工具箱和大规模数据集，还有深势科技官网提供的免费算力券，欢迎广大目标检测爱好者前来华山论剑！

竞赛详情/报名：[PRCV2024 SARDet](https://nb.bohrium.dp.tech/competitions/detail/6816082019?tab=introduce)

**Comment by zcablii** [comment]
Now, the Test set is available! Please check in the README☺️
